.github/workflows/lint-pr.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint PR
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          scopes: |
+            analyzer
+            api
+            api_client
+            installer
+            legacy
+            playout
+            ui
+            worker
+
```

Comment:
Is this intentional to allow blank scope?

Reply:
Nope, I wanted to split our project components  from abstract scope, but I just found out ci is actually a type.